### DISCUSSION TITLE
Welcome to iOS16-Live-Activities Discussions!

### DISCUSSION THREAD

**Discussion by 1998code** [1998code/iOS16-Live-Activities#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aravindk-foodhub** [comment]
Hi, Is there a way to show delivery time in minutes, like 14 mins and countdowns to 13 mins, 12 mins and so on.

**Reply by 1998code** [reply]
Hello @aravindk-foodhub,

According to the SwiftUI doc, there is no showSeconds option.

<img width="951" alt="CleanShot 2023-02-26 at 15 52 28@2x" src="https://user-images.githubusercontent.com/54872601/221399245-cd806036-d48a-40cb-b31d-768bb206443c.png">
